openshift/configmap-template.yaml

```diff
@@ -117,3 +119,15 @@ parameters:
   name: RSYSLOG_PORT
   required: true
   value: '10514'
+  
+- displayName: PROMETHEUS_PUSHGATEWAY_HOST
+  description: A host to start the pushgateway server.
```

Comment:
We are not starting the pushgateway. The description should state that the metrics will be pushed to this gateway, if the gateway configuration is provided.

Reply:
Okay, got it!